src/util/mapbox.js

```diff
@@ -183,7 +185,7 @@ const normalizeTileURL = function(tileURL: string, sourceURL?: ?string, tileSize
         urlObject.params.push(`sku=${skuToken}`);
     }
 
-    return makeAPIURL(urlObject);
+    return makeAPIURL(urlObject, customAccessToken);
```

Comment:
Line 182/184 above will not add the `sku` parameter to the url if `config.ACCESS_TOKEN` is not provided. With the per-Map access token option, it is foreseeable that someone uses it without setting `mapboxgl.accessToken`.

Reply:
`config.REQUIRE_ACCESS_TOKEN && (config.ACCESS_TOKEN || customAccessToken) && skuToken)` should work